### ISSUE TITLE
[Bugs] - CollectionView,When I use ScrollTo, VerticalOffset does not return to initial value in Android

### ISSUE THREAD

**Issue by vsfeedback** [xamarin/Xamarin.Forms#12418]
_This issue has been moved from [a ticket on Developer Community](https://developercommunity.visualstudio.com/content/problem/1208409/bugs-collectionviewwhen-i-use-scrollto-verticaloff.html)._

---
[severity:It bothers me. A fix would be nice]
Xamarin Forms 4.8.0.1451
Visual Studio 2019 16.7.5

1. Bug 1
When I use mycollectionview. ScrollTo (0) or mycollectionview. ScrollTo (0, animate: true), VerticalOffset increments its value
plus 1 each time I use the ScrollTo.

2. Bug 2
When I use mycollectionview. ScrollTo (0, animate: false) VerticalOffset does not return to initial value when
I use the ScrollTo.

Attached videos of the problems. Greetings

---
#### Feedback Bot on 10/5/2020, 11:16 AM: 

<p>We have directed your feedback to the appropriate engineering team for further evaluation. The team will review the feedback and notify you about the next steps.</p>

---
### Original Solutions
(no solutions)

**Comment by Cr15Denis** [comment]
1. Bug 1
When I use mycollectionview. ScrollTo (0) or mycollectionview. ScrollTo (0, animate: true), VerticalOffset increments its value
plus 1 each time I use the ScrollTo.

![Verticaloffset animation true andoridgif](https://user-images.githubusercontent.com/70722700/95498971-d54cf400-0961-11eb-89ae-c638a0fdc559.gif)

2. Bug 2
When I use mycollectionview. ScrollTo (0, animate: false) VerticalOffset does not return to initial value when
I use the ScrollTo.

![Verticaloffset animation false andoridgif](https://user-images.githubusercontent.com/70722700/95499103-05949280-0962-11eb-9b73-7692488a79d0.gif)

**Comment by jsuarezruiz** [comment]
@Cr15Denis Could you attach that sample or a small sample where reproduce the issue?. I would like to do tests and see how the `VerticalOffset` is being updated, etc.

**Comment by Cr15Denis** [comment]
of course, I share the project with you. https://github.com/Cr15Denis/Bugs-XamarinForms.git

**Comment by Cr15Denis** [comment]
similar problem #10966
